vanilla-framework/vanilla-framework
# Optimise SVG icons

## Done

Optimised the icon SVGs

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/examples/patterns/icons/icons-light/
- See that the icons are visually identical to the icons on https://docs.vanillaframework.io/en/patterns/icons

## Details

Fixes #1968
